server/api/routes/subscriptions.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const { createSubscription } = require('../actions/subscriptions');
+
+// Create a new router object, where we will have CRUD routes for our subscriptions table 
+// under the route /users/:userId/subscriptions/
+const router = express.Router();
+
+router.route('/:userId/subscriptions')
```

Comment:
per the other comment you won't need all of this:

```suggestion
router.route('/')
```